packages/block-template/templates/html/dev/index.html

```diff
@@ -1,49 +0,0 @@
-<!--
```

Comment:
This file is needed

Reply:
I see, thanks! Restored in https://github.com/blockprotocol/blockprotocol/pull/480/commits/3d42c37ee9ea22d4c63ecdf83ab97d7e5ef3a117, ignore files updated in https://github.com/blockprotocol/blockprotocol/pull/480/commits/834f5c0a93bd2081a38781e6e4977649948409a5